tests/test_unittest/test_gtc/test_oir.py

```diff
@@ -43,3 +53,50 @@ def test_no_horizontal_offset_allowed():
 def test_mask_must_be_bool():
     with pytest.raises(ValidationError, match=r".*must be.* bool.*"):
         HorizontalExecution(body=[], mask=DummyExpr(dtype=A_ARITHMETIC_TYPE)),
+
+
+def test_temporary_default_3d():
+    temp = Temporary(name="a", dtype=DataType.INT64)
+    assert temp.dimensions == (True, True, True)
+
+    temp1d = Temporary(name="b", dtype=DataType.INT64, dimensions=(True, False, False))
+    assert temp1d.dimensions == (True, False, False)
+
+
+def test_assign_to_ik_fwd():
```

Comment:
This is testing the shared root validator? I'd only test it once in test_common.py (as mentioned already above).